directives/annotated.py

```diff
@@ -373,3 +397,6 @@ def setup(app):
     app.add_node(altered_node, html=(visit_altered_node, depart_altered_node),
             latex=ignore_visitors)
     app.add_directive('altered-code-block', AlteredCodeBlock)
+
+    app.connect('builder-inited', add_assets)
```

Comment:
I thought that `add_assets` could be called directly in `setup` and using the `builder-inited` signal is unnecessary.

Reply:
I added a comment explaining why

Reply:
I agree now that the signals are required. The comment is important for explaining it.